### ISSUE TITLE
OpenSSL TypeError exception when starting REST interface

### ISSUE THREAD

**Issue by tssoffsec** [EmpireProject/Empire#579]
## Empire Version
Git clone 12.06.2017

## OS Information (Linux flavor, Python version)
Kali, Python 2.7.3

```
python empire --rest --restport 1337 --username empireadmin --password Password123
 
 * Starting Empire RESTful API on port: 1337
 * RESTful API token: bg4e81gni19dml9g64drn3h9qijpc0tom377jzb9
 * Running on https://0.0.0.0:1337/
Exception in thread Thread-1:
Traceback (most recent call last):
  File "/usr/lib/python2.7/threading.py", line 552, in __bootstrap_inner
    self.run()
  File "/home/epa1/Empire/lib/common/helpers.py", line 792, in __run
    self.__run_backup()
  File "/usr/lib/python2.7/threading.py", line 505, in run
    self.__target(*self.__args, **self.__kwargs)
  File "empire", line 1264, in thread_api
    start_restful_api(empireMenu=empireMenu, suppress=False, username=args.username, password=args.password, port=args.restport)
  File "empire", line 1224, in start_restful_api
    app.run(host='0.0.0.0', port=int(port), ssl_context=context, threaded=True)
  File "/usr/lib/python2.7/dist-packages/flask/app.py", line 703, in run
    run_simple(host, port, self, **options)
  File "/usr/lib/python2.7/dist-packages/werkzeug/serving.py", line 617, in run_simple
    inner()
  File "/usr/lib/python2.7/dist-packages/werkzeug/serving.py", line 599, in inner
    passthrough_errors, ssl_context).serve_forever()
  File "/usr/lib/python2.7/dist-packages/werkzeug/serving.py", line 402, in make_server
    passthrough_errors, ssl_context)
  File "/usr/lib/python2.7/dist-packages/werkzeug/serving.py", line 347, in __init__
    self.socket = tsafe.Connection(ssl_context, self.socket)
  File "/usr/lib/pymodules/python2.7/OpenSSL/tsafe.py", line 11, in __init__
    self._ssl_conn = apply(_ssl.Connection, args)
TypeError: Connection() argument 1 must be OpenSSL.SSL.Context, not tuple
```

**Comment by xorrior** [comment]
@tssoffsec Can you try a fresh git pull? This may or may not have been before some patches were in place to fix ssl support.

**Comment by xorrior** [comment]
@tssoffsec fixed in a6ae921
